examples/dmm/dmm.py

```diff
@@ -294,17 +294,19 @@ def main(args):
     # package repeated copies of val/test data for faster evaluation
     # (i.e. set us up for vectorization)
     def rep(x):
-        y = np.repeat(x, n_eval_samples, axis=0)
-        return y
+        rep_shape = torch.Size([x.size(0) * n_eval_samples]) + x.size()[1:]
```

Comment:
we would probably just use `vectorize_particles=True` now.... (needn't be in this PR)

Reply:
I didn't realize this was being used in that context, and probably tried being too clever coming up with a torch solution for this. Let us defer this to a separate PR.